### PR TITLE
feat(Button): add danger color and outline none

### PR DESCRIPTION
**Pull request by zouxuoz**

### CODE REVIEW

File: client/components/ui/forms/Button/Button.css [hunk 1]
```diff
@@ -33,6 +34,15 @@
   }
 }
 
+.button_color_danger {
+  background-color: var(--weakredColor);
```
Comment (rosendi) [inline client/components/ui/forms/Button/Button.css]: if it a danger color then call it `danger` color :-)

File: client/components/ui/forms/Button/Button.css [hunk 2]
```diff
@@ -33,6 +34,15 @@
...
+  color: #fff;
+
+  &:hover {
+    background-color: var(--paleredColor);
```
Comment (rosendi) [inline client/components/ui/forms/Button/Button.css]: use css3 function to darken or lighten the danger color.

File: client/components/ui/forms/Button/Button.css [hunk 3]
```diff
@@ -33,6 +34,15 @@
   }
 }
 
+.button_color_danger {
```
Comment (rosendi) [inline client/components/ui/forms/Button/Button.css]: In general that isn't a color modifier, i.e it's just a danger button. I understand that it changes only the color, but by the meaning it isn't only a color. May be it should be `button_danger`

File: client/components/ui/forms/Button/Button.css (hunk 1 above)
Comment (zouxuoz) [inline client/components/ui/forms/Button/Button.css]: fixed

File: client/components/ui/forms/Button/Button.css [hunk 4]
```diff
@@ -24,12 +25,21 @@
...
 
-.button_color_blue {
-  background-color: var(--dirtblueColor);
+.button_type_primary {
+  background-color: var(--primaryColor);
   color: #fff;
 
   &:hover {
-    background-color: var(--sickblueColor);
+    background-color: color(var(--primaryColor) whiteness(25%));
```
Comment (rosendi) [inline client/components/ui/forms/Button/Button.css]: May be just w(25%) ? :-)

**Comment by rosendi** [comment]
lgtm

**Comment by radiansz** [comment]
lgtm

**Pull request merged**